src/containers/flows/MintDropNFT/steps/Approval/Approval.tsx

```diff
@@ -133,7 +133,7 @@ const Approval: React.FC<ApprovalProps> = ({
 			<>
 				<span>
 					Approving WILD spending contract. This may take a few minutes. This
-					transaction must finish before you can continue to mint your Pet.
+					transaction must finish before you can continue to mint your Beast.
```

Comment:
I think we should change the `Beast` to `Beast(s)`, as it's possible to mint multiple.

Reply:
I don't think this step will kick in for current drop.